### PR TITLE
[Enhance] Add an example of pcd to bin format.

### PR DESCRIPTION
**Pull request by 274869388**
## Motivation

The motivation of this PR is to add an example of pcd to bin format file.

## Modification

- customize_datasets.md  
Added installation examples and code examples for converting pcd to bin files using the pypcd tool in this document.

## BC-breaking (Optional)

No.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Review by JingweiZhang12 — APPROVED** [review]

**Review by ZCMax — APPROVED** [review]

**Comment by ZCMax** [comment]
@ZwwWayne Ready to be merged

File: docs/en/advanced_guides/customize_dataset.md [hunk 1]
```diff
@@ -18,7 +18,30 @@ The ideal situation is that we can reorganize the customized raw data and conver
 
 Currently, we only support '.bin' format point cloud for training and inference. Before training on your own datasets, you need to convert your point cloud files with other formats to '.bin' files. The common point cloud data formats include `.pcd` and `.las`, we list some open-source tools for reference.
 
-1. Convert pcd to bin: https://github.com/leofansq/Tools_RosBag2KITTI
+1. Convert pcd to bin: https://github.com/DanielPollithy/pypcd
+
+- You can install pypcd with the following command:
+
+```bash
+pip install pytest-runner
```
Comment (JingweiZhang12) [inline docs/en/advanced_guides/customize_dataset.md:26]: Is `pytest-runner` necessary?

File: docs/en/advanced_guides/customize_dataset.md [hunk 2]
```diff
@@ -18,7 +18,30 @@ The ideal situation is that we can reorganize the customized raw data and conver
...
+```python
+from pypcd import pypcd
+
+pcd_data = pypcd.PointCloud.from_path('point_cloud_data.pcd')
+points = np.zeros([pcd_data.width, 4], dtype=np.float32)
+points[:, 0] = pcd_data.pc_data['x'].copy()
+points[:, 1] = pcd_data.pc_data['y'].copy()
+points[:, 2] = pcd_data.pc_data['z'].copy()
+points[:, 3] = pcd_data.pc_data['intensity'].copy().astype(np.float32)
+with open('point_cloud_data.bin', 'rb') as f:
```
Comment (JingweiZhang12) [inline docs/en/advanced_guides/customize_dataset.md:41]:
```suggestion
with open('point_cloud_data.bin', 'wb') as f:
```

File: docs/en/advanced_guides/customize_dataset.md (hunk 1 above)
Comment (274869388) [reply docs/en/advanced_guides/customize_dataset.md]: It's deleted.

File: docs/en/advanced_guides/customize_dataset.md (hunk 2 above)
Comment (274869388) [reply docs/en/advanced_guides/customize_dataset.md]: done

**Comment by JingweiZhang12** [comment]
LGTM. It's ready to be merged. @ZwwWayne

**Pull request merged**